frontend/src/metabase/components/Popover.jsx

```diff
@@ -21,11 +22,11 @@ export default class Popover extends Component {
     super(props, context);
 
     this.state = {
-      width: null,
-      height: null,
+      maxHeight: 0,
```

Comment:
A small comment here about why we have maxHeight would be helpful. My hope would be a little context might help prevent us from getting into a situation like we had with `width` and `height`

Reply:
Added a comment